.github/workflows/build-test-e2e.yml

```diff
@@ -63,6 +63,12 @@ jobs:
         with:
           name: ${{ env.BUILD_ARTIFACT_NAME }}
           path: ${{ env.BUILD_DIR }}
+      - name: Confirm build downloaded
+        run: |
+          if [ ! -f "/$(pwd)/${{ env.BUILD_DIR }}/${{ env.BUILD_NAME }}" ]; then
+            echo "::error ::downloaded WAR not found"
+            exit 1
+          fi
```

Comment:
The latest [e2e run](https://github.com/Sage-Bionetworks/SynapseWebClient/actions/runs/7200394434) on this branch did not repro the [failures](https://github.com/Sage-Bionetworks/SynapseWebClient/actions/runs/7198896769/job/19611941233) on develop, where the Playwright tests failed to run with this error: `TypeError: The argument 'file' cannot be empty. Received ''`. However, I can repro the error locally by deleting the SWC WAR, so that the file doesn't exist at the volume path specified when launching the SWC tomcat container. 

Add this check to confirm that the build was successfully downloaded to help with troubleshooting if the same error occurs on develop in the future.